packages/webapp/src/components/Consent/index.js

```diff
@@ -33,7 +33,7 @@ export default function PureConsent({ onSubmit, checkboxArgs, onGoBack, text, di
         style={{ width: '90%', overflowY: 'scroll' }}
```

Comment:
I noticed that the consent text is off centre. Should we remove `width: 100%` or centre the div?